src/core_plugins/kibana/index.js

```diff
@@ -40,9 +40,21 @@ module.exports = function (kibana) {
 
         injectVars: function (server, options) {
           let config = server.config();
+
+          //is URL the default? if setting to undefined has no effect, it means it reverted back to the default
+          //alternatively, consider adding this to the manifest
+          const configuredUrl = server.config().get('tilemap.url');
```

Comment:
Good point. I would assume so yes. I'd consider it an "override" by the Kibana-admin.